logsearchapi/server/partitions.go

```diff
@@ -242,3 +242,23 @@ func (c *DBClient) vacuumData(diskCapacityGBs int) {
 
 	}
 }
+
+func (c *DBClient) partitionTables() {
+	checkInterval := 12 * time.Hour
```

Comment:
make this configurable @krisis through deployment spec? - also using ticker is super tricky it is usually just okay to use time.Sleep() - in this case time.Sleep is enough

Reply:
The time period to check if next partition table is available is internal to logsearchapi.